src/auth/__tests__/index.spec.js

```diff
@@ -380,6 +427,19 @@ describe('auth', () => {
         });
         expect(fetchMock.lastCall()).toMatchSnapshot();
       });
+
+      it('should continue with custom parameters', async () => {
+        fetchMock.postOnce('https://samples.auth0.com/oauth/token', tokens);
+        expect.assertions(1);
+        await auth.loginWithSMS({
+          phoneNumber: '+5491159991000',
+          code: '123456',
+          audience: 'http://myapi.com',
+          scope: 'openid',
+          custom: 'should not be filtered',
+        });
+        expect(fetchMock.lastCall()).toMatchSnapshot();
+      });
```

Comment:
Should we add similar tests for `loginWithOTP()`, and `loginWithOOB()`, `loginWithRecoveryCode()` as well?